Shared/Samples/Create convex hull around points/CreateConvexHullAroundPointsView.swift

```diff
@@ -0,0 +1,122 @@
+    
+    /// A Boolean indicate whether the reset button can be pressed.
+    @State var resetIsDisabled = true
+    
+    var body: some View {
+        // Create a map view to display the map.
+        MapView(map: model.map, graphicsOverlays: model.graphicsOverlays)
+            .onSingleTapGesture { _, mapPoint in
+                model.inputPoints.append(mapPoint)
+                model.pointsGraphicsOverlay.addGraphic(Graphic(geometry: mapPoint, symbol: model.markerSymbol))
+                createIsDisabled = false
+                resetIsDisabled = false
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    // Create button.
+                    Button("Create") {
+                        model.createConvexHull()
+                        createIsDisabled = true
+                    }
+                    .disabled(createIsDisabled)
+                    // Reset button.
+                    Button("Reset") {
+                        model.reset()
+                        createIsDisabled = true
+                        resetIsDisabled = true
+                    }
+                    .disabled(resetIsDisabled)
+                }
+            }
+    }
+}
+
+private extension CreateConvexHullAroundPointsView {
+    // The view model for this sample.
+    private class Model: ObservableObject {
+        /// A map with a topographic basemap.
+        let map = Map(basemapStyle: .arcGISTopographic)
+        
+        /// An array of input points to be used in creating the convexHull.
+        var inputPoints: [Point] = []
+        
+        /// An array that contains the graphics overlays for the sample.
+        var graphicsOverlays: [GraphicsOverlay] {
+            [pointsGraphicsOverlay, convexHullGraphicsOverlay]
+        }
+        
+        /// The graphics overlay for the input points graphics.
+        let pointsGraphicsOverlay = GraphicsOverlay()
+        
+        /// The graphics overlay for the convex hull graphic.
+        private let convexHullGraphicsOverlay = GraphicsOverlay()
+        
+        /// A red simple marker symbol to display where the user tapped on the map.
+        let markerSymbol = SimpleMarkerSymbol(style: .circle, color: .red, size: 10)
+        
+        /// A blue simple line symbol for the outline of the convex hull graphic.
+        private let lineSymbol = SimpleLineSymbol(style: .solid, color: .blue, width: 4)
+        
+        /// A hollow polygon simple fill symbol for the convex hull graphic.
+        private lazy var fillSymbol = SimpleFillSymbol(style: .noFill, outline: lineSymbol)
+        
+        /// Reset the points and graphics.
+        func reset() {
+            inputPoints.removeAll()
+            pointsGraphicsOverlay.removeAllGraphics()
+            convexHullGraphicsOverlay.removeAllGraphics()
+        }
+        
+        /// Create the convex hull graphic using the inputPoints.
+        func createConvexHull() {
```

Comment:
Since it is not a factory method, I'm OK with the method name not using "make". [Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/) doesn't have requirement on this type of procedural methods.